jck/runtime.api/playlist.xml

```diff
@@ -1001,6 +1000,16 @@
 				<version>17</version>
 				<impl>ibm</impl>
 			</disable>
+			<disable>
+				<comment>Disabled on all platforms on 18 for backlog/issues/633 (intermittent machine issue). To be run manually</comment>
+				<version>18</version>
```

Comment:
Does it make sense to change line 1000 to `17+`?

Reply:
May be it made sense. Updated. We have to just remember now to to put this target in list of tests to run manually for all 17+ lists.

Reply:
another reason to echo disabled tests :)